### PR TITLE
Wording and design update on choogle show and modals

### PR DESCRIPTION
**Pull request by clement-bergantz**

### CODE REVIEW

File: app/assets/images/bg-wave.svg [hunk 1]
```diff
@@ -0,0 +1,16 @@
+
```
Comment (simong33) [inline app/assets/images/bg-wave.svg:1]: what kind of language is this ?? To format the image ?

File: app/assets/stylesheets/components/_alert.scss [hunk 2]
```diff
@@ -8,10 +8,10 @@
   color: white;
 }
 .alert-info {
-  background: $green;
```
Comment (simong33) [inline app/assets/stylesheets/components/_alert.scss:11]: Something wrong with Sass color variables?

File: app/assets/stylesheets/components/_tag.scss [hunk 3]
```diff
@@ -2,7 +2,7 @@
   color: white;
   border-radius: 8px;
   /*Background color in show page directly linked with the color in the DB*/
-  /*background-color: purple;*/
```
Comment (simong33) [inline app/assets/stylesheets/components/_tag.scss:5]: Some variables have a "$" some other not... Why? Just curious.

File: app/assets/stylesheets/config/Find Results [hunk 4]
```diff
@@ -0,0 +1,114 @@
+Searching 3321 files for "gray" (regex)
```
Comment (simong33) [inline app/assets/stylesheets/config/Find Results:1]: Should get rid of this file in .gitignore

File: app/assets/stylesheets/config/_bootstrap_variables.scss [hunk 5]
```diff
@@ -10,24 +10,24 @@ $body-bg:                 $beton;
...
-$brand-success:   $green;
-$brand-info:      $yellow;
-$brand-danger:    $red;
-$brand-warning:   $orange;
+$gray-base:       $darkbeton;
+$brand-primary:   $st-patricks-blue;
+$brand-success:   green;
+$brand-info:      yellow;
+$brand-danger:    red;
+$brand-warning:   orange;
```
Comment (simong33) [inline app/assets/stylesheets/config/_bootstrap_variables.scss:19]: Same questions for color variables notation with / without $

File: app/models/choogle.rb [hunk 6]
```diff
@@ -8,9 +8,9 @@ class Choogle < ApplicationRecord
   validates :slug, presence: true
 
   validate do |choogle|
-    choogle.errors.add(:base, "Your Choogle should have a title") if choogle.title.blank?
-    choogle.errors.add(:base, "You have to add an Event date") if choogle.happens_at.blank?
-    choogle.errors.add(:base, "You have to indicate a closing date for votes") if choogle.due_at.blank?
+    choogle.errors.add(:base, "Please add a title to your poll, so your friends understand what it is") if choogle.title.blank?
+    choogle.errors.add(:base, "Please tell your friends when the event will take place") if choogle.happens_at.blank?
+    choogle.errors.add(:base, "Please set a deadline for your poll") if choogle.due_at.blank?
   end
```
Comment (simong33) [inline app/models/choogle.rb:14]: Way better messages!

File: app/assets/images/bg-wave.svg (hunk 1 above)
Comment (clement-bergantz) [reply app/assets/images/bg-wave.svg:1]: SVG used for the background image. SVG is to graphics what HTML is to text

File: app/views/choogles/_choogle_show.html.erb [hunk 7]
```diff
@@ -89,15 +89,8 @@
...
         <ul class="media-list">
-          <li class="media-comment">
-             <%= image_tag('chooglebot.png', class: "avatar") %>
-            <div class="media-body">
-              <p class="media-heading">ChoogleBot
-              <div class='text-area'>Welcome to the ChoogleChat, human! Talk to your friends now 🚀 </div>
-              <small class="text-muted"> <%= Time.now.strftime("%^a %H:%M %p") %></small></p>
-            </div>
-          </li>
           <%= render @comments %>
```
Comment (simong33) [inline app/views/choogles/_choogle_show.html.erb:94]: I guess it's for the better. Bye bye Chooglebot!

File: app/assets/stylesheets/components/_alert.scss (hunk 2 above)
Comment (clement-bergantz) [reply app/assets/stylesheets/components/_alert.scss:11]: I removed all Sass color variables not in our chart

File: app/views/choogles/_choogle_show.html.erb [hunk 8]
```diff
@@ -107,15 +100,15 @@
     <!-- New notification input -->
     <div class="notification">
       <div class="notification-content">
-        <p>Enter your email to be notify when the timer is done</p>
+        <p>Get the selected place right in your inbox when the timer is done</p>
```
Comment (simong33) [inline app/views/choogles/_choogle_show.html.erb]: "when the timer is **up**" ?

File: app/assets/stylesheets/components/_tag.scss (hunk 3 above)
Comment (clement-bergantz) [reply app/assets/stylesheets/components/_tag.scss:5]: Actually, there should be no more purple (not in our chart) <span class="emoji-outer emoji-sizer"><span class="emoji-inner" style="background: url(chrome-extension://immhpnclomdloikkpcefncmfgjbkojmh/emoji-data/sheet_apple_64.png);background-position:56.25000000000001% 70.83333333333334%;background-size:4900%" data-codepoints="1f914"></span></span>
Main colors are built-in, so you don't need a Sass variable for them

File: app/views/choogles/_form.html.erb [hunk 9]
```diff
@@ -1,19 +1,17 @@
...
+      <div class="col-xs-12 col-sm-3 small-padding">
+        <%= f.input :happens_at, as: :text, label: false, placeholder: "When ?", input_html: { class: 'flatpickr-input form-home-input textarea' } %>
       </div>
-    </div>
-    <div class="row">
-      <div class="col-xs-12 col-sm-8">
-        <%= f.input :due_at, as: :text, label: "Closing votes date", placeholder: "Due date for voting", input_html: { class: 'flatpickr-input form-date-element'} %>
+      <div class="col-xs-12 col-sm-3 small-padding">
+        <%= f.input :due_at, as: :text, label: false, placeholder: "Deadline for the poll ?", input_html: { class: 'flatpickr-input form-home-input textarea'} %>
       </div>
```
Comment (simong33) [inline app/views/choogles/_form.html.erb:12]: Great placeholder! 10 times better than the last one

File: app/views/choogles/_form.html.erb [hunk 10]
```diff
@@ -1,19 +1,17 @@
...
-      <div class="col-xs-12 col-sm-4 date-form-btn btn-setup-1">
-        <%= f.button :submit, value: "Let's Choogle", class: "btn-choogle btn-margin" %>
+      <div class="col-xs-12 col-sm-2 small-padding">
+        <%= f.button :submit, value: "Create a poll", class: "btn-choogle-home" %>
       </div>
```
Comment (simong33) [inline app/views/choogles/_form.html.erb:15]: So no more "Let's Choogle", that's official. Fine by me!

File: app/views/pages/home.html.erb [hunk 11]
```diff
@@ -1,72 +1,106 @@
...
-      <div class="banner-head">
-        <h1>Choogle a place to go</h1>
-        <p>Decision-making tool to choose the best place to go</p>
-      </div>
-      <!-- form -->
-      <div class="date-form">
+  <div class="head">
+    <div class="head-content">
+      <h1>Where are<br>we going next ?</h1>
+      <h4>Choogle is the easiest way to collectivly choose a bar, a restaurant, an holiday destination or any other place</h4>
```
Comment (simong33) [inline app/views/pages/home.html.erb]: Great! 
Just I think it's "**a** holiday" -> https://www.a-or-an.com/a_an/holiday

File: app/views/pages/home.html.erb [hunk 12]
```diff
@@ -1,72 +1,106 @@
...
+                <h3>Make a collective decision</h3>
+                <p>Share your poll with your friends. They can add new places and vote for their favorites. No account needed.</p>
+              </div>
+            </div>
+          </div>
+          <div class="col-xs-12 col-sm-6">
+            <div class="headline">
+                <h2>⏰</h2>
+                <div class="headline-text">
+                  <h3>Stop losing time on long group decision making</h3>
```
Comment (simong33) [inline app/views/pages/home.html.erb]: decision-making

File: app/views/pages/home.html.erb [hunk 13]
```diff
@@ -1,72 +1,106 @@
...
+              <p>Upvoted by Amelia, Oliver, Jack, George, Ava, Henry, Emily and Jacob.</p>
+              <p>Proposed by Ava for the <strong>Annual festival trip.</strong></p>
+
+          </div>
+          <div id="caption-1" class="carousel-caption">
+              <h2>Rosa Bonheur Bar, Belleville, Paris</h2>
+              <div class="tag">
+                <h5 class="tags tags-color3">guinguette</h5><h5 class="tags tags-color1">metro 7</h5><h5 class="tags tags-color4">metro 11</h5><h5 class="tags tags-color7">belleville</h5><h5 class="tags tags-color2">outdoor</h5><h5 class="tags tags-color6">rosé</h5><h5 class="tags tags-color8">tapas</h5><h5 class="tags tags-color7">pizza</h5>
+              </div>
+              <p>Upvoted by Lisa, Romain, Eliott, Simon, Marie, Vincent, Tatiana and Sylvain.</p>
```
Comment (simong33) [inline app/views/pages/home.html.erb:76]: ;)

File: app/views/proposals/_form.html.erb [hunk 14]
```diff
@@ -5,15 +5,15 @@
   <% unless user_signed_in? || current_or_guest_user.first_name != 'guest' %>
     <%= f.simple_fields_for :user do |h| %>
       <%= h.input :first_name,
-      label: "Let your friends know who you are",
+      label: false,
       placeholder: "Your name",
```
Comment (simong33) [inline app/views/proposals/_form.html.erb:9]: Can't you just delete "label" ?

File: app/views/pages/home.html.erb (hunk 13 above)
Comment (clement-bergantz) [reply app/views/pages/home.html.erb:76]: That's a great "du dur" section :)

File: app/views/proposals/_form.html.erb [hunk 15]
```diff
@@ -5,15 +5,15 @@
...
       input_html: {id: 'sign_up_code',
       value: nil } %>
     <% end %>
   <% end %>
-  <%= f.input :place, label: "Add a new place" %>
-  <%= f.association :tags, class: 'tag-selector', collection: current_or_guest_user.tags, label: "Add tags to this place (max 20 char)" %>
-  <%= f.button :submit, value: "Add a new place", class: 'btn btn-choogle btn-proposal-margin' %>
+  <%= f.input :place, label: false, placeholder: "Search a place", error: "Please add a place"%>
+  <%= f.association :tags, class: 'tag-selector', collection: current_or_guest_user.tags, label: "Describe this place with your own words" %>
```
Comment (simong33) [inline app/views/proposals/_form.html.erb]: Ok, but the user has to get that we want **single** words. But let's try this anyway, it's a fine solution!

File: app/views/proposals/_form.html.erb [hunk 16]
```diff
@@ -24,7 +24,7 @@
       tags: true,
       theme: "bootstrap",
       maximumInputLength: 20,
-      placeholder: "Select or create your own tags."
+      placeholder: "cheap, outdoor, tapas, point of view..."
     }
```
Comment (simong33) [inline app/views/proposals/_form.html.erb:28]: that might do it!

**Review by simong33 — COMMENTED** [review]
Very hard work! All is fine by me. Just some wording, and typo correction.

File: app/assets/images/bg-wave.svg (hunk 1 above)
Comment (simong33) [reply app/assets/images/bg-wave.svg:1]: Ok good to know

File: app/assets/stylesheets/components/_alert.scss (hunk 2 above)
Comment (simong33) [reply app/assets/stylesheets/components/_alert.scss:11]: OK

File: app/assets/stylesheets/components/_tag.scss (hunk 3 above)
Comment (simong33) [reply app/assets/stylesheets/components/_tag.scss:5]: Like paradize-pink? Nice

File: app/views/pages/home.html.erb (hunk 13 above)
Comment (simong33) [reply app/views/pages/home.html.erb:76]: I was exactly commenting this. Du dur is great for this section of the site. Too many variables leans to a Russian roulette home page

File: app/views/proposals/_form.html.erb (hunk 15 above)
Comment (clement-bergantz) [reply app/views/proposals/_form.html.erb]: Humm, your right I forget that they can understand this as a free text input with sentences. I propose "describe this place with tags". I think we should mention that tags are related to the specific place and not the event. The better solution would be to have tags-like formated placeholders.

**Pull request merged**